EDDI/MainWindow.xaml.cs

```diff
@@ -565,43 +576,85 @@ private void ChangeLog_Click(object sender, RoutedEventArgs e)
             changeLog.Show();
         }
 
-        public static void uploadLog(IProgress<string> progress)
+        public static void prepareLog(IProgress<string> progress)
         {
-            using (WebClient client = new WebClient())
+            try
             {
-                try
+                string IssueLogDir = Constants.DATA_DIR + @"\logexport\";
```

Comment:
The naming convention is that variables start with lower case, so "issueLogDir" etc